### DISCUSSION TITLE
Columnar Selection

### DISCUSSION THREAD

**Discussion by edermachado** [pulsar-edit/.github#137] [General]
Hello,
In fact, it's a question: how do I make the columnar selection in Pulsar? I tried several ways and I couldn't, any suggestions?
Thanks

**Comment by savetheclocktower** [comment]
**Editor: Add Selection Above** and **Editor: Add Selection Below** will add a new cursor above/below the current one. On my platform (macOS) those are bound to <kbd>Ctrl-Shift-Up</kbd> and <kbd>Ctrl-Shift-Down</kbd> by default.

You can also hold <kbd>Cmd</kbd> (most likely <kbd>Ctrl</kbd> in other environments) and click anywhere in the editor to add a new cursor.

If you want to click and drag to create a rectangular selection, there's a package for that called [sublime-style-column-selection](https://web.pulsar-edit.dev/packages/sublime-style-column-selection).

**Reply by edermachado** [reply]
Thank you very much! I understood the working logic of the selection in columns! And I also installed the suggested package! It looks great! Hugs!
